include/boost/json/impl/monotonic_resource.ipp

```diff
@@ -10,61 +11,110 @@
 #ifndef BOOST_JSON_IMPL_MONOTONIC_RESOURCE_IPP
 #define BOOST_JSON_IMPL_MONOTONIC_RESOURCE_IPP
 
+#include <algorithm>
```

Comment:
std includes should come last. Rationale: Placing them last makes it more likely for a compiler error to occur if an include is missing.

Reply:
why do we need `<algorithm>` that's an expensive header

Reply:
`std::max`

Reply:
its a shame to have to include that just for one piss-ant function

Reply:
It's gone now